gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1091,22 +1094,22 @@ void gnc_file_ofx_import (GtkWindow *parent)
         selected_filename = conv_name;
 #endif
 
-        DEBUG("Opening selected file");
+        DEBUG("Opening selected file(s)");
         num_trans_processed = 0;
-        libofx_proc_file(libofx_context, selected_filename, AUTODETECT);
-        // Now would be a good time to see whether the view has anything in it!
+        for (iter = selected_filenames; iter; iter = iter->next)
+            libofx_proc_file (libofx_context, iter->data, AUTODETECT);
```

Comment:
Should we not be doing the conv_name procedure on all the file names in the list?

Reply:
You're right.